ckan/config/who.ini

```diff
@@ -1,7 +1,7 @@
-[plugin:auth_tkt]
-use = ckan.lib.auth_tkt:make_plugin
-# If no secret key is defined here, beaker.session.secret will be used
-#secret = somesecret
+[plugin:use_beaker]
+use = repoze.who.plugins.use_beaker:make_plugin
+key_name = ckan_session
+delete_on_logout = True
 
 [plugin:friendlyform]
```

Comment:
I think `rememberer_name = auth_tkt` also needs to be replaced here with `rememberer_name = use_beaker`